manifests/init.pp

```diff
@@ -56,17 +56,20 @@
   include php::apache
 
   include php::cli
-  # php module wants to use apt as the provider but the package isn't available on ubuntu 12.04.
-  class { 'php::extension::redis':
-    provider => 'pecl',
```

Comment:
You will want to keep this class and just change the provider to apt and the package to php5-redis.